slave/process-coffeeroom/changelog

```diff
@@ -0,0 +1,5 @@
+perun-slave-process-coffeeroom (1.0.0) stable; urgency=low
```

Comment:
Please start package version at 3.1.0 , it's related to perun version. We are in a transition to more proper versioning through perun.